Main/Source/nonhuman.cpp

```diff
@@ -959,10 +959,11 @@ truth vampirebat::SpecialBiteEffect(character* Victim, v2 HitPos, int BodyPartIn
     return false;
 }
 
-bool ChameleonPolymorphRandomly(chameleon* c){
-  if(!c->StateIsActivated(POLYMORPH_LOCK)){
-    character* NewForm = c->PolymorphRandomly(100, 1000, 500 + RAND() % 500);
-    if(NewForm==NULL)ABORT("chameleon PolymorphRandomly failed"); //means needs more checks
+bool ChamaleonPolymorphRandomly(chameleon* c){
+  character* NewForm = c->PolymorphRandomly(100, 1000, 500 + RAND() % 500);
+
+  if(NewForm != NULL)
+  {
```

Comment:
The conflict got resolved into this. 👅

Reply:
Oh no :sob: Sorry @iology, can you please fix? (This is really the last time I use that resolve conflicts feature, I think I'm going to write to GitHub).

Reply:
I rebased the branch.